### PR TITLE
- [FP-2773]: Scene - opening a scene the info tab on the right is open and empty

### PR DESCRIPTION
**Pull request by quirinpa**
- [FP-2773](https://movai.atlassian.net/browse/FP-2773): Scene - opening a scene the info tab on the right is open and empty
  - Depends on:
    - https://github.com/MOV-AI/frontend-npm-lib-scene/pull/282
    - https://github.com/MOV-AI/frontend-npm-ide/pull/596
    
 
[Gravação de ecrã a partir de 31-07-2024 13:52:25.webm](https://github.com/user-attachments/assets/5edcd29c-b530-4b3f-a675-42f8ef74fa41)

[FP-2773]: https://movai.atlassian.net/browse/FP-2773?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by diasdm — CHANGES_REQUESTED** [review]
@quirinpa  version 1.2.2 has been released, add a `# TDB` at the top of the changelog pls

File: src/decorators/withBookmarks.jsx [hunk 1]
```diff
@@ -34,12 +34,11 @@ const withBookmarks = Component => {
...
-    const firstKey = useMemo(() => Object.keys(bookmarks)[0] ?? false, [bookmarks]);
     const [active, setActive] = useState();
-    const renderedView = useMemo(
-      () => bookmarks[active || firstKey]?.view || [],
-      [active, bookmarks, firstKey]
-    );
+    const renderedView = useMemo(() => {
+      const firstKey = Object.keys(bookmarks)[0] ?? false;
+      return bookmarks[active || firstKey]?.view || [];
+    }, [active, bookmarks, firstKey]);
```
Comment (manuelnogueiramov) [inline src/decorators/withBookmarks.jsx:41]: How is this not throwing errors? You're setting `firstKey` in the dependency array whilst is now being created inside the useMemo... That means that `firstKey` doesn't exist outside of the function, how does this not trigger all sorts of errors? Did you test this?

**Review by manuelnogueiramov — CHANGES_REQUESTED** [review]

File: src/decorators/withBookmarks.jsx (hunk 1 above)
Comment (quirinpa) [reply src/decorators/withBookmarks.jsx]: I did this in the meaning and did not get to test it. I thought David would grab it next.
Sorry.

**Review by manuelnogueiramov — DISMISSED** [review]
Codewise looks better :+1: 
I didn't test this though. If @diasdm says it's good, I'm good :)

**Comment by diasdm** [comment]
Replaced by https://github.com/MOV-AI/frontend-npm-lib-scene/pull/294 and https://github.com/MOV-AI/frontend-npm-ide/pull/596

**Pull request closed without merging**